R/antigen.garnish_predict.R

```diff
@@ -189,26 +189,35 @@ parallel::mclapply(dt[, spc %>% unique], function(s){
   # run blastp-short for iedb matches
     message("Running blastp for iedb homology...")
 
-    if (file.exists("Ms_nmer_fasta.fa") & file.exists("Hu_nmer_fasta.fa"))
-            system("cat Ms_nmer_fasta.fa Hu_nmer_fasta.fa > iedb_query.fa")
+    if (file.exists("Ms_nmer_fasta.fa"))
 
-    if (list.files(pattern = "nmer_fasta") %>% length == 1)
-        file.copy(list.files(pattern = "nmer_fasta"), to = "iedb_query.fa")
+      system(paste0(
+        "blastp -query Ms_nmer_fasta.fa -task blastp-short -db ~/antigen.garnish/Mu_iedb.bdb -evalue 100000000 -matrix BLOSUM62 -gapopen 11 -gapextend 1 -out iedbout_mu.csv -num_threads ", parallel::detectCores(),
```

Comment:
would it be possible to document the meaning of these flags inline here, for clarity for blastp calls?

Reply:
Sure, you mean as comments inline?

Reply:
yes

Reply:
Done